packages/ruby/index.ts

```diff
@@ -47,30 +47,29 @@ async function bundleInstall(
 ) {
   debug(`running "bundle install --deployment"...`);
   const bundleAppConfig = await getWriteableDirectory();
+  const { exitCode, stderr, stdout } = await execa(
+    bundlePath,
+    ['install', '--deployment', '--gemfile', gemfilePath, '--path', bundleDir],
+    {
+      stdio: 'pipe',
+      reject: false,
+      env: {
+        ...process.env,
```

Comment:
Very nice

Reply:
Subtle change but I think it was a bug to not include the env vars. Didn't make a difference in this case.